code/modules/mafia/controller.dm

```diff
@@ -326,6 +326,10 @@
 	QDEL_LIST(all_roles)
 	turn = 0
 	votes = list()
+	//map gen does not deal with landmarks
+	for(var/i in landmarks)
```

Comment:
QDEL_LIST(landmarks)

Reply:
if true i need to also adjust another part of the code